crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -964,10 +988,25 @@ type MergeCoinsTransaction {
 	coins: [TransactionArgument!]!
 }
 
+"""
+The basic four abilities for a move object
```

Comment:
These are also the only abilities possible for a move object, right? At least for now. How about something like, "Abilities are keywords in Sui Move that define how types behave at the compiler level. There are four abilities in Sui Move today."

Reply:
Yep agreed, these are just the abilities, not the "basic" abilities.  I like @wlmyng's suggested change here, but would also drop the "There are four abilities in Sui Move today." part -- that should be obvious from looking at the type, and it's a safe assumption that reference docs describe how things are today.

Reply:
Excellent, thanks. I tend to add adjectives to everything, so thanks for getting me back on track.